### DISCUSSION TITLE
Remote backend?

### DISCUSSION THREAD

**Discussion by bellwether-softworks** [emilk/egui#3292] [Q&A] [answered]
I've noted that egui sports a long list of backends, but all of them that I've seen assume the app resides on the same system as the renderer.  I'm curious whether it's feasible to have a streaming backend that renders in a remote interface (e.g. wasm-based renderer, standalone app), akin to Dioxus's [Liveview](https://dioxuslabs.com/learn/0.4/getting_started/liveview) or a [Blazor app](https://learn.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-7.0&WT.mc_id=dotnet-35129-website#blazor-server).

FWIW, I haven't looked at the throughput yet for tessellation in egui for a typical app, so don't know whether sending raw triangles is cheaper versus video streaming, and I'm also somewhat curious what impact network latency would have on the render loop.  At this point, I'm asking the question more out of idle curiosity than any strong need.

**Comment by emilk** [comment, marked as answer]
I've actually experimented a bit with this in https://github.com/emilk/eterm - it is quite a few version behind though.

**Reply by bellwether-softworks** [reply]
I think this is exactly what I was looking for; glad to get confirmation that the necessary bandwidth isn't a major issue.
